dmutils/content_loader.py

```diff
@@ -265,50 +264,24 @@ def get_error_messages(self, errors, lot):
         :return: error dictionary with human readable error messages
         """
         errors_map = {}
-        for question_id, message_key in errors.items():
-            field_name = question_id
-            if question_id == 'serviceTypes':
-                field_name = question_id = 'serviceType{}'.format(lot)
-            elif question_id in PRICE_FIELDS:
-                message_key = self._rewrite_pricing_error_key(question_id, message_key)
-                field_name = question_id = 'priceString'
-            elif message_key == 'assurance_required':
-                field_name = '{}--assurance'.format(question_id)
-
-            validation_message = self.get_error_message(question_id, message_key)
-
-            errors_map[field_name] = {
+        for field_name, message_key in errors.items():
+            if field_name == 'serviceTypes':
+                field_name = 'serviceType{}'.format(lot)
```

Comment:
I've just had a dig through the API and the rendered frontend and it looks like you're right. All I can think is that error handling for service types has been broken since June.